util/dummy-login/index.ts

```diff
@@ -55,31 +55,37 @@ const check: LoginCheck = async ({ userid, password }) => {
             username: userid,
             displayName: user.displayName,
             roles: user.roles.concat(["ROLE_ANONYMOUS", "ROLE_USER"]),
+            email: user.email,
         }
         : "forbidden";
 };
 
 const DUMMY_PASSWORD = "tobira";
-const DUMMY_USERS: Record<string, { displayName: string; roles: string[] }> = {
+const DUMMY_USERS: Record<string, { displayName: string; roles: string[]; email: string; }> = {
     "admin": {
         displayName: "Administrator",
         roles: ["ROLE_ADMIN", "ROLE_USER_ADMIN", "ROLE_SUDO"],
+        email: "admin@example.org",
     },
     "sabine": {
         displayName: "Sabine Rudolfs",
         roles: ["ROLE_USER_SABINE", "ROLE_INSTRUCTOR", "ROLE_TOBIRA_MODERATOR"],
+        email: "sabine@example.org",
     },
     "björk": {
         displayName: "Prof. Björk Guðmundsdóttir",
         roles: ["ROLE_USER_BJÖRK", "ROLE_EXTERNAL", "ROLE_TOBIRA_MODERATOR"],
+        email: "björk@example.org",
```

Comment:
Hu, TIL that non-ASCII characters encoded as UTF8 are proposed to being valid in email addresses. But it seems like it is not implemented everywhere yet. I mean... it probably doesn't matter but maybe use `bjoerk` here? But :shrug: not too important